src/wallet/scriptpubkeyman.cpp

```diff
@@ -280,6 +280,47 @@ bool LegacyScriptPubKeyMan::GetReservedDestination(const OutputType type, bool i
     return true;
 }
 
+bool LegacyScriptPubKeyMan::TopUpInactiveHDChain(const CKeyID seed_id, int64_t index, bool internal)
```

Comment:
0715613

Too much work for parameterizing `TopUp` with `seed_id` or other reasons?

Reply:
Needing `index` makes it harder to parameterize.